website/content/release-notes/_index.md

```diff
@@ -21,8 +21,8 @@ Chores:
 - Dev-env: Override GOBIN for inv dev-env ([PR 2219](https://github.com/metallb/metallb/pull/2219))
 - Metrics: add ipv4/6 addresses_in_use_total and addresses_total ([PR 2151](https://github.com/metallb/metallb/pull/2151))
 - Squash the prefixes in FRR mode, avoiding duplicate prefixes in the FRR configuration ([PR 2234](https://github.com/metallb/metallb/pull/2234))
-
 - BGP: move the matching peer logic one level up ([PR 2233](https://github.com/metallb/metallb/pull/2233))
+- Remove support for legacy AddressPools ([PR 2252](https://github.com/metallb/metallb/pull/2252))
```

Comment:
Can you be a bit more verbose here on what we are removing? Also, I'd move it to the features section so it's more evident.

Reply:
done